BusinessLayer/SongControl/SongControl.cpp

```diff
@@ -1,4 +1,5 @@
 #include "SongControl.h"
+#include <QDir>
```

Comment:
Place this include above the other.
In general, we should be styling it like:
`#include <Libraries>`
`#include "MyHeaders`

In alphabetical order